soccer/Robot.cpp

```diff
@@ -630,6 +629,7 @@ void OurRobot::replanIfNeeded(const Geometry2d::CompositeShape& global_obstacles
 		//  TODO: This is Stupid. This should be fixed in the RRT planner or the Bezier Algorithm.
 		if (_path->destination() && (*_path->destination() - dest).mag() > 0.025) {
 			_pathInvalidated = true;
+			addText("Destination Moved");
```

Comment:
It doesn't show up often for a play like `LineUp`, but if you run something with the `Mark` behavior, it does something like

~~~{.py}
target = (opponent.pos - ourgoal) * factor + ourgoal
robot.move_to(opponent.pos - ourgoal
~~~

In this case, it'll show up pretty often.